app/containers/Main/index.js

```diff
@@ -71,11 +72,14 @@ class Main extends Component {
   }
 
   configureAccountKit = () => {
+    const phoneNumber = RNSimData.getTelephoneNumber();
+    const countryCode = RNSimData.getCountryCode();
+
     AccountKit.configure({
-      countryWhitelist: [ 'ID' ],
-      defaultCountry: 'ID',
+      countryWhitelist: ['ID'],
+      defaultCountry: countryCode.toUpperCase() || 'ID',
       initialPhoneCountryPrefix: '+62',
-      initialPhoneNumber: '87809000750'
+      initialPhoneNumber: phoneNumber || '87809000750'
```

Comment:
hm, can you add the default value to empty string instead of the above random number?

Reply:
Sure. Give me a minute